js/content/community.js

```diff
@@ -2741,6 +2742,73 @@ let FriendsPageClass = (function(){
         sortFriends(SyncedStorage.get("sortfriendsby"));
     };
 
+    FriendsPageClass.prototype.addPoster = function() {
+        let manage = document.querySelectorAll("#manage_friends");
+        if (manage.length === 0) { return; }
+
+        let commentArea = `<div class="row commentthread_entry" style="background-color: initial; padding-right: 24px;">
+                <div class="commentthread_entry_quotebox">
+                    <textarea rows="3" class="commentthread_textarea" id="comment_textarea" placeholder="${Localization.str.add_comment}" style="overflow: hidden; height: 20px;"></textarea>
+                </div>
+                <div class="commentthread_entry_submitlink">
+                    <a class="btn_grey_black btn_small_thin" id="FormattingHelpPopup">
+                        <span>Formatting help</span>
+                    </a>
+                    <span class="emoticon_container">
+                        <span class="emoticon_button small" id="emoticonbtn">
+                        </span>
+                    </span>
+                    <span class="btn_green_white_innerfade btn_small" id="comment_submit">
+                        <span>${Localization.str.friends_commenter}</span>
+                    </span>
+                </div>
+            </div>
+            <div class="row" id="log">
+                <span id="log_head"></span>
+                <span id="log_body"></span>
+            </div>`
+
+        HTML.beforeBegin("#manage_friends > div:nth-child(3)", commentArea);
+        ExtensionLayer.runInPageContext(`function() {
+            let delay = 7000;
+            ToggleManageFriends();
+            new CEmoticonPopup($J('#emoticonbtn'), $J('#comment_textarea'));
+            $J("#comment_submit").click(() => {
+                const total = $J(".selected").length;
+                const msg = $J("#comment_textarea").val();
+                if (total === 0 || msg.length === 0) {
+                    ShowAlertDialog("${Localization.str.alert}", "${Localization.str.friends_commenter_warning}");
+                    return;
+                }
+
+                if(/([a-zA-Z0-9]+:\/\/)?([a-zA-Z0-9_]+:[a-zA-Z0-9_]+@)?([a-zA-Z0-9.-]+\.[A-Za-z]{2,4})(:[0-9]+)?(\/.*)?/.test(msg)) {
```

Comment:
You need to double escape the forward slash, otherwise it's a syntax error

Reply:
Works fine here:
![image](https://user-images.githubusercontent.com/4411977/68532560-f4d93800-031e-11ea-8cff-0b0ee6062881.png)

Reply:
Good catch